### ISSUE TITLE
Duplicate repos for the same library?

### ISSUE THREAD

**Issue by DRSDavidSoft** [libtorrent/libtorrent#17]
@arvidn 

Can you please help me understand, why there are **two** repos for `libtorrent-rasterbar`?
* https://github.com/arvidn/libtorrent
* https://github.com/libtorrent/libtorrent

This is causing confusion for me :/

**Comment by arvidn** [comment]
well, the reason it happened was a friend of mine set up a git mirror of the subversion repository a long time ago. then it was forgotten (and I didn't think I had access to it). Then I transitioned from subversion to github, so now there were two. One of them quite stale.

Now I do have access to it, but I haven't done anything with it. moving from `arvidn` -> `libtorrent` seems like a big and involved transition, so I'm hesitant to do that lightly.

**Comment by DRSDavidSoft** [comment]
Thanks for explaining!

Mind I suggest that If you do have access now on this repo, you could the parent libtorrent account into an "organization" account, join it and then give your own account (@arvidn) full access to the repo.
That way, in the future you can commit directly into libtorrent/libtorrent instead of arvidn/libtorrent.

In any case, thanks for all your work on libtorrent!

P.S. does libtorrent have an icon or a logo?

**Comment by arvidn** [comment]
yeah, I have commit access to the repo. I'm hesitant to move development there because of other contributor needing to switch repo, and users referring to the current repo.

I suppose I should at least add a redirection for now.

there isn't really a logo for libtorrent. I've given in a try once or twice, but I'm not a designer, so it didn't turn out very well. There was a guy who offered a logo, which is in the repo now, but I haven't really embraced it. I'm hoping to have a simpler logo with some symbolism in it, but it's not easy :)

**Comment by cas--** [comment]
@arvidn fyi the url needs fixing from github.org to .com
